wordpress-mobile/AztecEditor-iOS
# Prepare release 1.16.0

Preparing release 1.16.0

To test:
 - Build the demo
 - Smoke test the demo content
 - Check if changelog and specs are correct.
